portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase

```diff
@@ -113,6 +113,82 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-10257 | Test Summary: Assert total amount of views in all cards"
+	@ignore = "true"
+	@priority = "4"
+	test AssertTotalAmountOfViewsInAllCards {
+		ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+		task ("Add a new user in DXP") {
+			JSONUser.addUser(
+				userEmailAddress = "ac@liferay.com",
+				userFirstName = "ac",
+				userLastName = "ac",
+				userScreenName = "ac");
+		}
+
+		task ("Sync Contact Data") {
+			ACDXPSettings.syncNewContactData();
+		}
+
+		task ("Login new user and view page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "ac@liferay.com",
+				userLoginFullName = "ac ac");
+		}
+
+		task ("Interation with page") {
+			ACUtils.navigateToSitePageWithUser(
+				pageName = "AC Page",
+				siteName = "Site Name");
+
+			ACUtils.closeAllSessionsAndWait();
+		}
+
+		task ("AC launch") {
+			ACUtils.launchAC();
+
+			ACProperties.switchProperty(propertyName = "${propertyName}");
+		}
+
+		task ("Asset views for 24 hours") {
+			ACNavigation.goToSites();
+
+			ACNavigation.switchTab(tabName = "Pages");
+
+			ACTimeFilter.clickTimeFilterButton();
+
+			ACTimeFilter.setLast24Hours();
+
+			ACNavigation.openItem(itemName = "AC Page - Site Name - ${dataSourceName}");
```

Comment:
Hey @calvinctvo, I think you forgot to use `StringUtil` to be able to create the `dataSourceName` that will be used in your macro. What do you think?

`var dataSourceName = StringUtil.extractFirst("${assignedPropertyName}", " ");`

Reply:
Hey @calvinctvo, I think that only this change was missing, this macro will get the correct name of the page you accessed in DXP to use in AC

Reply:
Hey @calvinctvo, I think you need to put a space inside the double quotes because `extractFirst` will extract everything from the string before the space.